conductor/agent/agent.py

```diff
@@ -0,0 +1,99 @@
+
+import subprocess
+import random
+
+from cocaine.services import Service, Locator
+
+from tornado.ioloop import IOLoop
+from tornado import gen
+
+import os
+
+
+agent_id = random.randint(1, 10**9)
+
+conductor = Service("conductor")
+
+
+@gen.coroutine
+def dump_stream():
+
+    ch = yield conductor.subscribe(agent_id)
+
+    while True:
+        rq = yield ch.rx.get()
+        print "================"
+        print "got request", rq
+
+        yield process_request(rq)
+
+            
+
+@gen.coroutine
+def process_request(rq0):
+
+    rq = rq0[1]
```

Comment:
what about ```rq0[0]```?
Seems it's better to change the signature to 
```python
def process_request(something, req)
``` 
if you need rq0[0], or don't pass it here